.gitignore

```diff
@@ -37,7 +37,7 @@ var/
 MANIFEST
 AUTHORS
 ChangeLog
-config/
+/config/
```

Comment:
Without this gitignore considers `config/` as a matching pattern to exclude, which means it will exclude any directory named `config` and not only the one in the git root.

By adding `/` before config, it will only exclude the `config` directory at the root and not `test/data/config` for example.

I mentioned this in the commit message. Please refer to the documentation for further information:
https://git-scm.com/docs/gitignore

Reply:
Yes, thanks!